.github/workflows/plugin-package.yml

```diff
@@ -49,6 +49,8 @@ jobs:
             pip download -r requirements.txt --only-binary :all: -d temp/
             unzip -o "temp/*.whl" -d pg_service_parser/libs
             rm -r temp
+            # To let QGIS know we've got subpackages in libs
+            touch pg_service_parser/libs/__init__.py
```

Comment:
what about adding it to the source by adding to .gitignore:
`!pg_service_parser/libs/__init__.py`

Reply:
Let's do that :+1: